client/templates/navbar/afNavbar.html

```diff
@@ -15,7 +15,7 @@
         <ul class="nav navbar-nav navbar-right">
           <form class="navbar-form navbar-left" role="search">
             <div class="form-group">
-              <input type="text" class="form-control" id='afSearchInput' placeholder="Search by Order Number/Id">
```

Comment:
@spencern are we saying we will never use Reaction ID for searching? Just shopify iD?